### PR TITLE
add favicon and attribution

### PR DESCRIPTION
**Pull request by emceeaich**
fix for issue #34

### CODE REVIEW

**Comment by emceeaich** [comment]
@mnoorenberghe or @mozfreddyb, can you review?

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
 # triage-center
+
+Tool for triage leads for Firefox-related components in Bugzilla to find bugs requring action. 
+
+Can be visited directly at https://mozilla.github.io/triage-center.
+
+To run locally, clone the repo, run `python process.py` in the directory and then use `python -m SimpleHTTPServer 8000` or your preferred method to bring up a static server.
+
+Uses an [SVG favicon](https://commons.wikimedia.org/wiki/File:Green_bug_and_broom.svg), provided under the GLGPL by https://commons.wikimedia.org/wiki/User:Poznaniak.
```
Comment (mozfreddyb) [inline README.md]: Nit: `Wikimedia User [Poznaniak](https://commons.wikimedia.org/wiki/User:Poznaniak)`, maybe?

File: index.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
   <title>Build Triage Links for Bugzilla Component(s)</title>
   <link rel="stylesheet" type="text/css" href="jquery-ui-1.11.4/jquery-ui.min.css">
   <link rel="stylesheet" type="text/css" href="index.css">
+  <link rel="icon shortcut" href="Green_bug_and_broom.svg" type="image/svg">
```
Comment (mozfreddyb) [inline index.html]: Firefox [seems](http://caniuse.com/#feat=link-icon-svg) to be the only major browser that allows SVG favicons. How much do we care for triage-center?

Comment (emceeaich) [inline index.html]: Since this is intended for Firefox and Bugzilla, I don't think that's a big issue.

**Pull request merged**